packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -4829,6 +4830,12 @@ mj_create_new_partitions(
   delete [] reduce_array;
 #endif
 
+  // the last member is utility used for atomically inserting the values.
+  // Sorting here avoids potential indeterminancy in the partitioning results
+  auto track_on_cuts_sort = Kokkos::subview(track_on_cuts,
+    std::pair<mj_lno_t, mj_lno_t>(0, track_on_cuts.size() - 1));
```

Comment:
`Kokkos::subview` takes a half-exclusive range `(start, end+1)`.  Also, what if `track_on_cuts.size()` is zero?